default.nix

```diff
@@ -0,0 +1,21 @@
+with import <nixpkgs> {};
+
+stdenv.mkDerivation {
+  name = "bee-docs";
+
+  buildInputs = [
+    coreutils diffutils
+    bash-completion less
+    gitFull
+    nodejs
+  ];
+
+  buildCommand = ''
+    npm run build
+  '';
+
+  shellHook =
+  ''
+    npm start
+  '';
+}
```

Comment:
Can you explain to me what this is?

Reply:
@Eknir ouch, it's a mistake. it shouldn't have been included, i'll clear this up in a moment.

meanwhile, the answer is here: https://github.com/ethersphere/docs.github.io/pull/119